src/wasm/wasm-emscripten.cpp

```diff
@@ -513,16 +513,25 @@ void AsmConstWalker::visitCall(Call* curr) {
     auto baseSig = getSig(curr);
     auto sig = fixupNameWithSig(curr->target, baseSig);
     auto* arg = curr->operands[0];
-    // The argument may be a get, in which case, the last set in this basic block
-    // has the value.
     if (auto* get = arg->dynCast<GetLocal>()) {
+      // The argument may be a local.get, in which case, the last set in this
+      // basic block has the value.
       auto* set = sets[get->index];
       if (set) {
         assert(set->index == get->index);
         arg = set->value;
       }
+    } else if (auto* value = arg->dynCast<Binary>()) {
+      // If the argument is an i32.add then assume its adding a constand to the
```

Comment:
typos: constand, additiona

Reply:
pedantry nit typo: s/its/it's/
In this case it is "it is" (it's) instead of "belonging to it" (its)